src/applications/lgy/coe/containers/App.jsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
-
+import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
 export default function App({ location, children }) {
   return (
-    <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-      {children}
-    </RoutedSavableApp>
+    <>
+      <Breadcrumbs>
+        <a href="/">Home</a>
+        {/* this will get updated when this route is added */}
+        <a href="/housing-assistance/">Housing assistance</a>
+        <a href="/housing-assistance/home-loans/">VA-backed home loans</a>
+        <a href="/housing-assistance/home-loans/apply-lgy-form-1880/">
+          Apply for Certificate of Eligibility Form 26-1880
```

Comment:
The breadcrumb should match the `h1` exactly. Also. not a fan of `lgy` in the URL... does that stand for "legacy"?

Reply:
It stands for Loan Guaranty, a VA line of business. I believe this URL was given to us by Content and IA. Though, it may be prone to change later on.